x-pack/plugins/security/public/session/session_timeout.test.tsx

```diff
@@ -147,6 +148,27 @@ describe('Session Timeout', () => {
       expect(close).toHaveBeenCalled();
       expect(cleanup).toHaveBeenCalled();
     });
+
+    test(`stop works properly for large timeouts`, async () => {
+      http.fetch.mockResolvedValue({
+        ...defaultSessionInfo,
+        idleTimeoutExpiration: now + 5_000_000_000,
```

Comment:
TIL you can use underscores in JS numbers 😁  pretty cool

Reply:
Yeah, I like it as well. Unfortunately we cannot use it in a non-test code yet (at least something was complaining about that during the build) :slightly_frowning_face: Should probably be easy to tweak our build/transpilation pipeline though, I see it's [`Stage 4` proposal already](https://github.com/tc39/proposal-numeric-separator).